### DISCUSSION TITLE
Uso de byte/sbyte (Linguagem de Programação com C#: Aula 11 - Byte

### DISCUSSION THREAD

**Discussion by rkmiecik** [balta-io/forum#686] [Tira Dúvidas] [answered]
Digamos que eu possuo a necessidade de ter um número para cada estado brasileiro, de 1 a 27, de acordo com o que está gravado no meu database. No vídeo você menciona que não é comum criar um byte em c#. Não entendi isto. Pois não deveria ser algo trivial eu optar por um byte quando possuo poucas opções possíveis, considerando o contexto que eu explicitei acima?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rkmiecik, como vai?

O tipo `byte` tem seu uso em diversos cenários, porém temos tipos específicos para números no C#, que tratam N fatores relacionados.

Você pode usar o `short` neste... e o motivo é que vai simplificar sua vida... uma soma, subtração, ou qualquer aritmética é mais fácil nos tipos numéricos.

Nada impede de você usar um `byte` também, depende muito do que você precisa... Só cuidado para não ficar "procurando pelo em ovo"
